src/options.c

```diff
@@ -548,7 +548,7 @@ void wlm_opt_parse(ctx_t * ctx, int argc, char ** argv) {
                     wlm_log_error("options::parse(): invalid empty title\n");
                     if (is_cli_args) wlm_exit_fail(ctx);
                 } else {
-                    ctx->opt.window_title = argv[1];
+                    ctx->opt.window_title = strdup(argv[1]);
```

Comment:
We need to free the old window title in this case so that it doesn't leak every time we set a new title via `--stream`.

```suggestion
                    free(ctx->opt.window_title);
                    ctx->opt.window_title = strdup(argv[1]);
```

The title should also be free'd in `wlm_cleanup_opt()` at line 28:

```c
void wlm_cleanup_opt(ctx_t * ctx) {
    if (ctx->opt.output != NULL) free(ctx->opt.output);
    if (ctx->opt.fullscreen_output != NULL) free(ctx ->opt.fullscreen_output);
    if (ctx->opt.window_title != NULL) free(ctx->opt.window_title);
}
```